data/json/monsters.json

```diff
@@ -3070,13 +3070,34 @@
         {
             "type" : "bite",
             "cooldown" : 10,
-            "damage_max_instance" : {
-                "type" : "stab",
-                "amount" : 10,
-                "armor_multiplier" : 0.2
-            }
+            "damage_max_instance" : [
+                { "damage_type" : "stab", "amount" : 10, "armor_multiplier" : 0.2 }
+            ]
         },
-        ["IMPALE", 25]
```

Comment:
Would it work if the JSON was defined separately from the critter?
That is, there would be "monster_attacks.json" defining the more complex attack and the monsters would just reference entries in it, similar to how `requirements` are defined.
This would prevent the situation we have with recipes right now, where many recipes specify a giant (longer than this attack's definition) lists of fruit or whatever.